src/attr/item.rs

```diff
@@ -119,14 +126,20 @@ impl ItemAttr {
 			}
 		}
 
-		// Delayed parsing of `skip_inner` to get access to all traits to be
+		// Delayed parsing of `skip_inner` and `incomparable` to get access to all traits to be
 		// implemented.
 		for meta in skip_inners {
 			self_
 				.skip_inner
 				.add_attribute(&self_.derive_wheres, None, &meta)?;
```

Comment:
we could swap the order of `add_attribute` to match the other attributes

Reply:
What other attributes? In any case, I don't mind.

Reply:
e.g. default doesn't really matter though